README.md

```diff
@@ -1,30 +1,24 @@
-# Pants Build System
+# Pantsbuild
 
-Pants is a scalable build system for _monorepos_: codebases containing 
-multiple projects, often using multiple programming languages and frameworks, 
-in a single unified code repository.
+Pantsbuild is a fast, scalable, user-friendly build system for codebases of 
+all sizes. It's currently focused on Python, Shell, Docker, and Go, with support for 
+Java and Scala coming soon.
 
-Some noteworthy features include:
+Pantsbuild orchestrates the various tools and steps that process your source code into deployable software, including:
```

Comment:
I was kind of expecting this sentence to end with ".. but not limited to:" so when it didn't, it kind of stopped me for a second :P